draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1715,167 +1834,175 @@ scope.
 
 ## Multi-technology IP Link Setup
 
-As described in {{optical-path-computation}}, there is a one-to-one relationship
-between a multi-technology intra-domain IP link and its underlay optical
-tunnel.
+As described in {{optical-path-computation}}, there is a one-to-one
+relationship between a multi-technology intra-domain IP link and its
+underlay optical tunnel.
 
-Therefore, to setup a new multi-technology intra-domain IP link, the MDSC
-requires the O-PNC to setup the  optical tunnel (using either the WDM
-Tunnel model or the OTN Tunnel model, if the optional OTN switching
-is supported) within the optical network and to steer the client
-traffic between the two cross-technology Ethernet links over that optical tunnel,
-using either the Ethernet Client Signal Model (for frame-based
-transport) or the Transparent CBR Client Signal Model (for
-transparent transport).
+Therefore, to set up a new multi-technology intra-domain IP link,
+the MDSC requires the O-PNC to set up the optical tunnel (using either
+the DWDM Tunnel model or the OTN Tunnel model, if optional OTN switching
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
the WDM Tunnel model or the OTN Tunnel model, if optional OTN switching
```